src/test/java/UnitTest.java

```diff
@@ -0,0 +1,179 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class UnitTest {
+    private StringEvaluation stringEvaluation;
+    private String testInput = null;
+    private ByteArrayOutputStream outputStreamCaptor;
+    private InputStream in;
+
+    @BeforeEach
+    protected void setUp(){
+        stringEvaluation = new StringEvaluation();
```

Comment:
Typically in unit testing we create test classes for each class e.g. StringEvaluationTest and StringExerciseMenuTest. Then each test class tests the functionality of each class so StringExerciseMenuTest would only test the functionality of the menu, mocking any interaction with StringEvaluation.